frontend/survey-project/src/app/components/shared/ui/button/basic-button.component.css.map

```diff
@@ -0,0 +1 @@
+{"version":3,"sources":["basic-button.component.scss","basic-button.component.css"],"names":[],"mappings":"AAAA;EACE,YAAA;EACA,eAAA;EACA,2BAAA;ACCF;ADCE;EACE,kBAAA;EACA,kBAAA;EACA,eAAA;ACCJ;ADCI;EACE,sCAAA;ACCN;ADCM;EACE,kCAAA;EACA,sBAAA;ACCR;ADGI;EACE,wCAAA;ACDN;ADGM;EACE,kCAAA;EACA,sBAAA;ACDR;ADKI;EACE,sCAAA;EACA,yBAAA;ACHN;ADKM;EACE,mCAAA;EACA,sBAAA;ACHR;ADOI;EACE,qCAAA;EACA,yBAAA;ACLN;ADOM;EACE,gCAAA;EACA,sBAAA;ACLR;ADSI;EACE,mCAAA;EACA,yBAAA;ACPN;ADSM;EACE,iCAAA;EACA,sBAAA;ACPR;ADWI;EACE,oCAAA;EACA,yBAAA;ACTN;ADWM;EACE,iCAAA;EACA,sBAAA;ACTR;ADaI;EACE,oCAAA;ACXN;ADaM;EACE,oCAAA;EACA,sBAAA;ACXR;ADeI;EACE,oCAAA;EACA,yBAAA;ACbN;ADeM;EACE,kCAAA;EACA,sBAAA;ACbR;ADiBI;EACE,eAAA;EACA,eAAA;ACfN;ADkBI;EACE,eAAA;EACA,gBAAA;AChBN;ADmBI;EACE,eAAA;EACA,gBAAA;ACjBN;ADoBI;EACE,WAAA;EACA,iBAAA;EACA,mCAAA;EACA,4BAAA;EACA,kCAAA;AClBN;ADqBI;EACE,cAAA;EACA,WAAA;EACA,iBAAA;EACA,mCAAA;EACA,4BAAA;EACA,kCAAA;ACnBN","file":"basic-button.component.css"}
```

Comment:
Not sure what is this used for, but I think it's not needed, we're using scss not css and the Angular builder will build our scss to css.